### DISCUSSION TITLE
Navigation with Arrow keys on Dropdown menu jumps to Cursor

### DISCUSSION THREAD

**Discussion by NotGumballer91** [ppy/osu#22989] [Q&A] [answered]
As the title says:
![SPZouEO](https://user-images.githubusercontent.com/64581009/227802600-a3d08a09-4cc3-490d-adea-728b541f661e.gif)
It works fine when the cursor not hovering over it. It's a weird behavior.

**Comment by Joehuu** [comment, marked as answer]
I noticed this in https://github.com/ppy/osu-framework/pull/5664#discussion_r1127072056 while fixing a different but related issue. Will revisit that fix soon.
